ormolu-live/www/editor.css

```diff
@@ -0,0 +1,38 @@
+.editor {
+    width: 100%;
+    display: inline-flex;
+    gap: 10px;
+    font-family: monospace;
+    line-height: 21px;
+    background: #282a3a;
+    border-radius: 2px;
+    padding: 20px 10px;
+}
+
+.editor > textarea {
+    flex-grow: 1;
+    line-height: 21px;
+    font-size: 14px;
+    overflow-y: hidden;
+    padding: 0;
+    border: 0;
+    background: #282a3a;
+    color: #FFF;
+    outline: none;
+    resize: none;
+}
+
+.line-numbers {
+    width: 20px;
+    text-align: right;
+}
+
+.line-numbers span {
+    counter-increment: linenumber;
+}
+
+.line-numbers span::before {
+    content: counter(linenumber);
+    display: block;
+    color: #506882;
+}
```

Comment:
Nitpick: add a trailing newline for consistency?

Reply:
IMO, the line numbers are always useful, regardless of how long the output is. How else is a user supposed to find a particular line with an error?
Also, please note that copy formatted output option was introduced https://github.com/tweag/ormolu/pull/918, and it’s important to not break that functionality. That is, copied output must not contain the line numbers because that’d make it uncompilable code.

Reply:
This doesn't impact the copy functionality I added - that copies the text from the output `div`, whereas this adds line numbers beside the input `textarea`, so it shouldn't have any impact. 👍